qutebrowser/config/config.py

```diff
@@ -657,6 +672,50 @@ def set_command(self, win_id, section_=None, option=None, value=None,
             message.info(win_id, "{} {} = {}".format(
                 section_, option, val), immediately=True)
 
+    @cmdutils.register(instance='config')
+    def reload_config(self):
+        """Reload the config & keyconfig files."""
+        old_config = objreg.get('config')
+        objreg.delete('config')
+        try:
+            _init_main_config(QApplication.instance())
+        # Restore the old config if we failed
+        except:
```

Comment:
Never ever use a bare `except`. Catch the exceptions which you expect (i.e. the one handled in `_init_main_config` with `reload=False` probably?) and then also display some appropriate error.